typescript/sdk/src/index.ts

```diff
@@ -639,3 +645,5 @@ export {
   WarpTypedTransaction,
 } from './warp/types.js';
 export { WarpCore, WarpCoreOptions } from './warp/WarpCore.js';
+
```

Comment:
When you leave empty spaces here, it interferes with the VS-code and/or prettier auto formatting